gcloud-java-core/src/main/java/com/google/gcloud/ServiceFactory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud;
+
+import java.io.Serializable;
+
+public interface ServiceFactory<
+        ServiceT extends Service, 
+        OptionsT extends ServiceOptions<?, OptionsT>>
+    extends Serializable {
```

Comment:
You are right, indeed. I am not a huge fan of serializable and far from an expert  but Effective Java 2 also states:
> [...] if a class or interface exists primarily to participate in a framework that requires all
participants to implement Serializable, then it makes perfect sense for the class
or interface to implement or extend Serializable

This is to say that my idea here was to extend `Serializable` to "force" all factories to be serializable as well.